packages/transition-backend/src/api/__tests__/public.routes.test.ts

```diff
@@ -825,6 +825,14 @@ describe('Testing API endpoints', () => {
         });
     });
 
+    test('/api/v1/version endpoint', async () => {
+
+        const response = await request(app).get('/api/versions');
+
+        expect(response.status).toStrictEqual(200);
+        expect(response.body).toStrictEqual(['1.1']);
```

Comment:
Do we want to mock the version or we want to change the test with each version bump?

Reply:
we want to be aware of a version change and make sure it is correctly returned, so it's hard-coded. I'll add a comment.